frontend/src/utils/enums.js

```diff
@@ -0,0 +1,6 @@
+let roleEnum = {
+  ADMIN: "ADMIN",
+  VOLUNTEER: "VOLUNTEER"

```

Comment:
Awesome, good idea - let's add PENDING to this as well so when we merge then @arynh can use this for deciding to render the landing page.